interface/wx/dcsvg.h

```diff
@@ -40,9 +40,10 @@ class wxSVGFileDC : public wxDC
 public:
     /**
         Initializes a wxSVGFileDC with the given @a f filename with the given
-        @a Width and @a Height at @a dpi resolution.
+        @a Width and @a Height at @a dpi resolution, and an optional @a title.
```

Comment:
Would be nice to explain how is the title used, at least to me (who doesn't know much about SVG) this is not really clear.